src/containers/WorkspaceDetails.tsx

```diff
@@ -92,15 +93,15 @@ class WorkspaceDetailsContainer extends React.PureComponent<Props> {
       <WorkspaceDetails
         ref={this.workspaceDetailsPageRef}
         workspacesLink={workspacesLink}
-        onSave={(workspace: che.Workspace) => this.onSave(workspace)}
+        onSave={(workspace: Workspace) => this.onSave(workspace)}
         history={this.props.history}
       />
     );
   }
 
-  async onSave(newWorkspaceObj: che.Workspace): Promise<void> {
+  async onSave(newWorkspaceObj: Workspace): Promise<void> {
```

Comment:
I think there might be a bit of an issue here, I just uncommented https://github.com/eclipse/che-dashboard/blob/master/src/components/DevfileEditor/index.tsx#L166 and then tried to see what object was passed into onSave for devworkspaces and it doesn't seem like a Workspace object is being passed in. Instead I get an object like:
```
{
    devfile: {},
    workspace: {}
}
```
which then eventually errors at https://github.com/eclipse/che-dashboard/blob/19480de5db23f72e42bd3a0ef198a38960f69870/src/store/Workspaces/index.ts#L239 because `workspace.ref` is undefined

Reply:
@JPinkney thanks for the feedback, I'll provide the fix.